doc/source/news/15.md

```diff
@@ -1,21 +1,33 @@
 # News - 15 series
 
-(release-15-14)=
-## Release 15.14 - 2025-07-30
+(release-15-16)=
+## Release 15.16 - 2025-09-03
```

Comment:
It's because we have already used '15.15' version.
https://github.com/mroonga/mroonga/commit/f5dc5723a36a30f1974a182d176b7d4d78665861